app/views/homepage/_more_on_govuk.html.erb

```diff
@@ -0,0 +1,52 @@
+<section class="homepage-more-on-govuk">
+  <%= render "govuk_publishing_components/components/heading", {
+    text: t('homepage.index.more'),
+    margin_bottom: 6,
+    font_size: "m",
+  } %>
+  <%= render "govuk_publishing_components/components/heading", {
+    text: t('homepage.index.most_active'),
+    margin_bottom: 4,
```

Comment:
Bit of a nitpick here but I think the translate strings (`homepage.index.more` and `homepage.index.most_active` are wrapped in double quotes elsewhere.